### DISCUSSION TITLE
MkDocs built not showing images

### DISCUSSION THREAD

**Discussion by biofsphere** [mkdocs/mkdocs#2754] [Q&A] [answered]
For some reason, mkdocs built does not show images. While in the localhost, it all looks perfectly fine. However, after building the documentation and firing up the index.html (ie, opening in a browser) all the images do not show. I noticed that if I edit the images' path in the generated html from "/img/name_of_img_file.jpg" to "../img/name_of_img_file.jpg" (pls, note the two dots) I get the images to load. However, I don't want to edit all html pages. That would not make sense for a static page generator. What am I doing wrong, pls?

FYI a clipping of the structure of the code and the built are attached, along with the list of the dependencies.

![dependencies](https://user-images.githubusercontent.com/92161383/150990651-7d607e77-c586-46d0-9c88-1d101aeccd05.jpeg)
![built structure](https://user-images.githubusercontent.com/92161383/150990654-063a52b3-586f-4be0-80d5-73dfe1858f7b.jpeg)
![code structure](https://user-images.githubusercontent.com/92161383/150990656-8c0d7772-44ab-405a-9bf1-c8b57d0a40ce.jpeg)

**Comment by Andre601** [comment, marked as answer]
`/img/name_of_img_file.jpg` is an absolute path while `../img/name_of_img_file.jpg` is a relative one.

My personal guess here is, that you host your docs under a sub-page (https://example.com/subpage) and not a subdomain (https://subdomain.example.com/) alone, which would explain the issue.

A absolute URL would in your case expect https://example.com/img/name_of_img_file.jpg to be where the file is located while the relative one is based on the location of where the URL is used in.

So, if my understanding is correct would for example a URL of https://example.com/pages/pages.html with a relative URL result in https://example.com/pages/img/name_of_img_file.jpg

With all that said, I would recommend updating the absolute URLs to include any sub-page names you have the docs under.
So if your URL would be https://example.com/docs should the URLs be `/docs/img/name_of_img_file.jpg`

I hope this somehow helps you

**Reply by biofsphere** [reply]
Spot on!! Thank you so much. Solved by making a subdomain instead.
